SSA/Projects/LLVMRiscV/Pipeline/const.lean

```diff
@@ -0,0 +1,2233 @@
+-- AUTOGENERATED Lean file: Constant lowering for RISC-V li instructions
+
+import SSA.Projects.LLVMRiscV.PeepholeRefine
+import SSA.Projects.LLVMRiscV.simpproc
+import SSA.Projects.RISCV64.Tactic.SimpRiscV
+
+ open LLVMRiscV
+
+    def li_riscv_n50 := [LV| {
+    ^entry ():
+      %0 = "li"() {imm = -50 : !i64}  : (!i64) -> (!i64)
+      %1 = "builtin.unrealized_conversion_cast"(%0) : (!i64) -> (i64)
+      llvm.return %1 : i64
+  }]
+def const_llvm_n50 : Com  LLVMPlusRiscV [] .pure (.llvm (.bitvec 64))  := [LV| {
+    ^entry ():
+      %1 = llvm.mlir.constant (-50) : i64
+      llvm.return %1 : i64
+  }]
+def llvm_const_lower_riscv_lin50 : LLVMPeepholeRewriteRefine 64 [] :=
+  {lhs:= const_llvm_n50, rhs:= li_riscv_n50,
```

Comment:
```suggestion
  {lhs := const_llvm_n50, rhs := li_riscv_n50,
```
mind your whitespaces around `:=`.

NIT: You could also consider using `where` instead of `{ ... }`, but I leave that choice up to you